Mathlib/Tactic/Algebraize.lean

```diff
@@ -176,7 +176,12 @@ def addProperties (t : Array Expr) : TacticM Unit := withMainContext do
     -- If it has, `p` will either be the name of the corresponding `Algebra` property, or a
     -- lemma/constructor.
     | some p =>
-      let cinfo ← try getConstInfo p catch _ => return
+      let cinfo ← try getConstInfo p catch _ =>
+        logWarning m!"Hypothesis {decl.toExpr} has type {decl.type}.\n\
```

Comment:
```suggestion
        logWarning m!"Hypothesis {decl.toExpr} has type{indentD decl.type}.\n\
```
I think that this is more standard.  If you like this, then you should also amend the test file.